CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+# Libero Community Code of Conduct
+
+The community has elected to adopt the Contributor Convenant in order to foster an open and welcoming environment where contributors can feel safe to make contributions in any way they wish.
+
+The full text of the code of conduct is below. Instances of abusive, harassing, or otherwise unacceptable behaviour may be
+reported by contacting our code of conduct representative, David Moulton from eLife (d.moulton@elifesciences.org).
+
+If you have further questions about this code of conduct please raise an issue on this repository or contact the team on [the Libero Community Slack](https://libero-community.slack.com).
+
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or
+  advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
```

Comment:
Again this lacks details on how the term "inappropriate" is defined and who / how its going to be judged.